app/components/TransactionFilters.tsx

```diff
@@ -30,7 +30,6 @@ type CheckableFilters = Pick<
     | TransactionKindString.TransferToEncrypted
     | TransactionKindString.TransferToPublic
     | TransactionKindString.FinalizationReward
-    | TransactionKindString.BakingReward
     | TransactionKindString.BlockReward
```

Comment:
I think all of them were replaced by the new staking reward thing.
But I don't know how it should be handled in the "transition" phase.

Reply:
Grouped them!